src/fragments/start/getting-started/vue/setup.mdx

```diff
@@ -248,7 +115,47 @@ import awsExports from './aws-exports';
 Amplify.configure(awsExports);
 ```
 
-And that's all it takes to configure Amplify. As you add or remove categories and make updates to your backend configuration using the Amplify CLI, the configuration in __aws-exports.js__ will update automatically.
+## Vue Vite Config
 
-Now that our Vue app is set up and Amplify is initialized, we're ready to add an API in the next step.
+When working with a Vue `create-app` that uses [Vite](https://vitejs.dev) you must make a few additional modifications. Please follow the steps below.
+
+**1.** Add the following script tag to the `index.html` file at the bottom of the `<body>` tag.
 
+```html
+...
+  <script>
+        window.global = window;
+        var exports = {};
+  </script>
+</body>
+```
+
+**2.** Update the `vite.config.ts` and add in a resolve object inside the `defineConfig({})` as seen below.
+
+```js
+...
+export default defineConfig({
+  plugins: [vue()],
+  resolve: {
+      alias: [
+      {
+        find: './runtimeConfig',
+        replacement: './runtimeConfig.browser',
+      },
+    ]
+  }
+})
```

Comment:
Nice!!!! This lines up with what we are recommending to customers as well - it will be nice to have a section in our docs we can refer customers to. I am probably going to take this part and add it to the troubleshooting side of the JS library guide. https://github.com/kevinold/repro-vite-amplify/commit/4d6b42291dbbcc9cee08f0ec52b416efd5ed7145

Reply:
OK great!